src/args.rs

```diff
@@ -0,0 +1,21 @@
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "monolith")]
+pub struct AppArgs {
+    #[structopt(short, long)]
+    pub url: String,
+
+    #[structopt(short, long)]
+    pub image: bool,
+
+    #[structopt(short, long)]
+    pub js: bool,
+
+    #[structopt(
+        short,
+        long,
+        default_value = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:66.0) Gecko/20100101 Firefox/66.0"
+    )]
+    pub agent: String,
+}
```

Comment:
I really like argument definition being in a separate file, but I sort of miss more verbose full option/flag names that clap had ("--no-js", "--user-agent", etc).
Could you please move changes related to argument processing into a separate branch so that we could refine it more and test how it behaves when compared to monolith using clap without it blocking any other changes?